public/video-ui/src/util/parseGridMetadata.js

```diff
@@ -0,0 +1,33 @@
+function parseMimeType(mimeType) {
+
+  //Normalise Mime Types coming from the grid.
+  switch(mimeType) {
+    case "jpg":
+      return "image/jpeg";
+    case "png":
+      return "image/png";
+  }
+
+  return mimeType;
+}
+
+
+function parseAsset(asset) {
+  return {
+    url: asset.secureUrl,
+    mimeType: parseMimeType(asset.mimeType),
+    size: asset.size,
```

Comment:
I wouldn't add it this side... these assets shouldn't be YouTube specific, but should probably be added into the youtube upload script